### PR TITLE
Make Host & Intern Dashboards Responsive

### LINKED ISSUE #313: Make dashboards mobile friendly
This is for intern, host and client

**Wireframes here: https://www.figma.com/file/SFarIViFgfYqrCfF5xOgLR/PressPad-Mobile-Wireframes?node-id=19%3A475**

### LINKED ISSUE #424: missing data in host dashboard
next intern section show `at the N/A`

image

### PR DESCRIPTION
**Pull request by yosefanajjar**
#313 dashboards mobile
### What I did?
- I added media queries for the pages and  other needed components
- I added more styles and removed unneeded/unused ones 
- I found some duplicate code which I removed but there are still more which would be removed in a future sprint, I did the most important things for the current sprint 
- I have also fixed ESlint errors that I found on my way

Please run the pages and let me know if there are any improvements that could be done

closes #313 
closes #424

### CODE REVIEW

File: client/src/components/Pages/InternDashboard/BookingSection.js [hunk 1]
```diff
@@ -64,16 +68,15 @@ export default function BookingSectionWrapper(props) {
...
             {firstBooking ? (
               <HiText>
-                {" "}
-                Hi {name.split(" ")[0]}, your host is expecting you {timeString}
-                .
+                Hi {name.split(" ")[0]}, your host is expecting you
+                <BoldTitle>{timeString} </BoldTitle>.
               </HiText>
             ) : (
               <HiText> Hi {name.split(" ")[0]}, .</HiText>
```
Comment (ashatat) [inline client/src/components/Pages/InternDashboard/BookingSection.js]:
```suggestion
             <HiText>
                Hi{" "}
                <span style={{ textTransform: "capitalize" }}>
                  {name.split(" ")[0]}.
                </span>
              </HiText>
```

File: client/src/components/Pages/InternDashboard/BookingSection.js [hunk 2]
```diff
@@ -64,16 +68,15 @@ export default function BookingSectionWrapper(props) {
...
             />
           </Col>
-          <Col span={20}>
+          <Col xs={18} sm={20}>
             {firstBooking ? (
               <HiText>
-                {" "}
-                Hi {name.split(" ")[0]}, your host is expecting you {timeString}
-                .
+                Hi {name.split(" ")[0]}, your host is expecting you
```
Comment (ashatat) [inline client/src/components/Pages/InternDashboard/BookingSection.js]:
```suggestion
                Hi{" "}
                <span style={{ textTransform: "capitalize" }}>
                  {name.split(" ")[0]}
                </span>
                , your host is expecting you
```

**Comment by thejoefriel** [comment]
Also need to sort out alignment of the view map so it's not overlapping the picture

image

**Comment by yosefanajjar** [comment]
On the links under the bio, it was `space-between` and I changed it into `space-around` and to me, it looks better than before now, I am open to any changes though

**Comment by yosefanajjar** [comment]
closes #313 
closes #424

**Pull request merged**